src/mcts/params.cc

```diff
@@ -146,6 +146,10 @@ const OptionId SearchParams::kOutOfOrderEvalId{
     "in the cache or is terminal, evaluate it right away without sending the "
     "batch to the NN. When off, this may only happen with the very first node "
     "of a batch; when on, this can happen with any node."};
+const OptionId SearchParams::kSyzygyFastPlayId{
+    "syzygy-fast-play", "SyzygyFastPlay",
+    "With DTZ tablebase files, only allow the network pick from winning moves "
+    "that have shortest DTZ to play faster (but not necessarily optimally)."};
```

Comment:
I don't think the option description is accurate. Maybe something like: "With DTZ tablebase files, avoid some of the winning moves that lead to long move sequences."

Reply:
The existing description looks accurate enough to me.  Maybe could be 'to often take less moves and play faster in winning positions (but still often not optimally).' if we want to be more verbose.